web/templates/user_info_popover_content.hbs

```diff
@@ -1,172 +1,191 @@
+                                {{else if status_emoji_info.url}}
+                                <img src="{{status_emoji_info.url}}" class="emoji status-emoji" />
+                                {{else}}
+                                <div class="emoji status-emoji emoji-{{status_emoji_info.emoji_code}}"></div>
+                                {{/if}}
+                                {{/if}}
+                                <span class="status_text">
+                                    {{status_text}}
+                                    {{#if is_me}}
+                                    <span class="clear_status_button">(<a tabindex="0" class="clear_status">{{t "clear"
+                                            }}</a>)</span>
+                                    {{/if}}
+                                </span>
+                            </span>
+                        </li>
+                        {{/if}}
 
-    {{#if is_me}}
-        <li>
-            <a tabindex="0" class="update_status_text">
-                <i class="fa fa-comments" aria-hidden="true"></i>
-                {{#if status_text}}
-                    {{t "Edit status" }}
-                {{else}}
-                    {{t "Set a status" }}
-                {{/if}}
-            </a>
-        </li>
-        {{#if invisible_mode}}
-        <li>
-            <a tabindex="0" class="invisible_mode_turn_off">
-                <i class="fa fa-circle-o" aria-hidden="true"></i> {{t "Turn off invisible mode" }}
-            </a>
-        </li>
-        {{else}}
-        <li>
-            <a tabindex="0" class="invisible_mode_turn_on">
-                <i class="fa fa-circle-o" aria-hidden="true"></i> {{t "Go invisible" }}
-            </a>
-        </li>
-        {{/if}}
-    {{/if}}
+                        {{#if is_me}}
+                        <li>
+                            <a tabindex="0" class="update_status_text">
+                                <i class="fa fa-comments" aria-hidden="true"></i>
+                                {{#if status_text}}
+                                {{t "Edit status" }}
+                                {{else}}
+                                {{t "Set a status" }}
+                                {{/if}}
+                            </a>
+                        </li>
+                        {{#if invisible_mode}}
+                        <li>
+                            <a tabindex="0" class="invisible_mode_turn_off">
+                                <i class="fa fa-circle-o" aria-hidden="true"></i> {{t "Turn off invisible mode" }}
+                            </a>
+                        </li>
+                        {{else}}
+                        <li>
+                            <a tabindex="0" class="invisible_mode_turn_on">
+                                <i class="fa fa-circle-o" aria-hidden="true"></i> {{t "Go invisible" }}
```

Comment:
Since we're updating the invisible mode icons to `zulip-icon-play-circle` and `zulip-icon-play-circle` in profile menu.
we would want to update the icons here as well.

Reply:
@ryanreh99, Here is the redesign PR if you want to take a look: #26170